templates/server-ha-active-service.yaml

```diff
@@ -15,6 +15,9 @@ metadata:
   annotations:
 {{ template "vault.service.annotations" .}}
 spec:
+  {{- if .Values.server.service.externalTrafficPolicy }}
+  externalTrafficPolicy: {{ .Values.server.service.externalTrafficPolicy | lower | title }}
```

Comment:
Are you certain that the extra template pipelines are necessary? I would say just take what is provided and let k8s sort it out.

Reply:
Since we have multiple `Services` I think it makes sense to have `externalTrafficPolicy` configurable per.

Reply:
@benashz I need to check what happens when one configures a service in K8S and uses a lower case value.
I suspect it might be failing to default in case of unknown value.
As per the services - you think this configuration might be necessary for the UI and the injector svcs also ? 
I'm trying not to overload the user with too many configurations.

Reply:
Checked it as shown below - the pipelines are indeed necessary in case the user enters local/cluster instead of Local/Cluster.
It depends on how we believe the user will be able to cope with filling the values as expected by K8S.

Reply:
I would prefer that we enforce that the user provides one of: `""`, `Cluster` or `Local`. This should be documented as the allowed values.